os/arch/arm/src/s5j/Make.defs

```diff
@@ -1,6 +1,6 @@
 ###########################################################################
 #
-# Copyright 2016 Samsung Electronics All Rights Reserved.
+# Copyright 2016-2017 Samsung Electronics All Rights Reserved.
```

Comment:
Don't modify this.

Reply:
Why? Done anyway.

Reply:
You can. but we don't need to update it whenever we modify somethings in this file.
It is enough to represent a year for the first time that copyright is created